mitodl/micromasters
# CMS: Added Semester Start Dates

#### What are the relevant tickets?
Fix #2808 

#### What's this PR do?
Allow staff to add future semester dates to program page.

#### How should this be manually tested?
Login as staff to /cms/. Start editing a program page, add few semester dates, click 'publish' and check the program page to see the changes.

#### Where should the reviewer start?
(Optional)

#### Any background context you want to provide?
The ui design in wagtail is a bit different. Originally I was trying to implement a read only field for semester title inside `FieldRowPanel`, but was not able to easily nest it in `InlinePanel` which allows the creation of a “cluster” of related objects to ProgramPage model.
Rob has approved this change.

#### Screenshots (if appropriate)
![screen shot 2017-03-15 at 1 09 07 pm](https://cloud.githubusercontent.com/assets/7574259/23961342/0495ab56-0981-11e7-97f0-0619af79fa2d.png)

![screen shot 2017-03-15 at 1 13 14 pm](https://cloud.githubusercontent.com/assets/7574259/23961361/10dbf35c-0981-11e7-832a-c1db127ddf96.png)


#### What GIF best describes this PR or how it makes you feel?
(Optional)
